src/gitlab.ts

```diff
@@ -118,9 +118,13 @@ export const runCommandInGitlabPipeline = async (ctx: Context, task: Task): Prom
   */
   let wasBranchRegistered = false;
   const waitForBranchMaxTries = 3;
-  const waitForBranchRetryDelay = 1024;
+  const waitForBranchRetryDelayMs = 2000;
```

Comment:
I'm mentally bothered that it is `2000` and no `2048` haha

But yeah, it's two seconds so it's a rounder number